saphanabootstrap-formula.spec

```diff
@@ -19,7 +19,7 @@
 # See also http://en.opensuse.org/openSUSE:Specfile_guidelines
 
 Name:           saphanabootstrap-formula
-Version:        0.2.9
+Version:        0.3.0
```

Comment:
Additionally, I wouldn't increase the version this way. I would update to `0.2.10` as this is just a bugfix or really minor update.

Reply:
yep